examples/dialog/Cargo.toml

```diff
@@ -17,7 +19,7 @@ bench = false
 test = false
 
 [dependencies]
-flipperzero = { version = "0.7.0", path = "../../crates/flipperzero", features = ["alloc"] }
-flipperzero-alloc = { version = "0.7.0", path = "../../crates/alloc" }
-flipperzero-sys = { version = "0.7.0", path = "../../crates/sys" }
-flipperzero-rt = { version = "0.7.0", path = "../../crates/rt" }
```

Comment:
Yup, this sounds reasonable. But I feel like these examples are already not trivially copy-pasteable (since they already use `path`) so the change is not that critical.

A better way to enhance this may be to create something like `cargo-generate` template which would be more customizable and easy to use, nut this surely is out of this PR's scope.

As for moving the examples under `flipperzero` directory -- it may be a good idea, but I would not hurry with this (since it can always be done under a separate task).

Reply:
Using `cargo-generate` was very much what I was thinking (e.g. [`esp-rs` uses this to good effect](https://github.com/esp-rs/esp-template)).

I might take a quick look at how difficult it would be to move the modules. If it turns out to be pretty easy to do, it should cut down the changes required for this PR.

Reply:
Amazing! In this case, will be waiting for updates on this :)